docs/source/basic/intro.rst

```diff
@@ -0,0 +1,90 @@
+Introduction
+============
+
+The *alpaka* library defines and implements an abstract interface for the *hierarchical redundant parallelism* model.
+This model exploits task- and data-parallelism as well as memory hierarchies at all levels of current multi-core architectures.
+This allows to achieve portability of performant codes across various types of accelerators by ignoring specific unsupported levels and utilizing only the ones supported on a specific accelerator.
+All hardware types (multi- and many-core CPUs, GPUs and other accelerators) are treated and can be programmed in the same way.
+The *alpaka* library provides back-ends for *CUDA*, *OpenMP*, *Boost.Fiber* and other methods.
+The policy-based C++ template interface provided allows for straightforward user-defined extension of the library to support other accelerators.
+
+The library name *alpaka* is an acronym standing for **A**\ bstraction **L**\ ibrary for **Pa**\ rallel **K**\ ernel **A**\ cceleration.
+
+Example
+-------
+
+.. literalinclude:: ../../../example/helloWorld/src/helloWorld.cpp
+   :language: C++
+   :caption: helloWorld.cpp
+
+.. code-block:: cmake
+   :caption: CMakeLists.txt
+
+   cmake_minimum_required(VERSION 3.15)
+
+   set(_TARGET_NAME helloWorld)
+   project(${_TARGET_NAME})
+
+   find_package(alpaka REQUIRED)
+
+   alpaka_add_executable(${_TARGET_NAME} helloWorld.cpp)
+   target_link_libraries(
+     ${_TARGET_NAME}
+     PUBLIC alpaka::alpaka)
+
+You can integrate alpaka into your project via ``find_package()`` in your ``CMakeLists.txt``.
+This requires, that you :doc:`install </basic/install>` alpaka.
+If you do not install alpaka in a default path such as ``/usr/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
+
+The cmake configuration decides which ``alpaka::acc`` is available during compiling. For example, if you configure your ``cmake`` build with the CUDA backend (``cmake -DALPAKA_ACC_GPU_CUDA_ENABLE=ON``), ``cmake`` checks, if the CUDA SDK is available and if it found, the C++ template ``alpaka::acc::AccGpuCudaRt`` is available during compiling.
```

Comment:
I would remove `cmake` from ``cmake -DALPAKA_ACC_GPU_CUDA_ENABLE=ON`` as this is not a complete command, and it's already clear we are talking about cmake arguments.